lib/topologies/replset.js

```diff
@@ -292,7 +292,7 @@ function connectNewServers(self, servers, callback) {
 
   // Handle events
   var _handleEvent = function(self, event) {
-    return function(err) {
+    return function(_, err) {
```

Comment:
why are we adding this spacer here?

Reply:
because I broke "BC" by adding the error as the first parameter emitted by the event rather than the topology. This keeps it from breaking anyone who might have been using core before, and requires less changes to the tests